src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlException.cs

```diff
@@ -14,7 +14,6 @@ namespace Microsoft.Data.SqlClient
 {
     /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlException.xml' path='docs/members[@name="SqlException"]/SqlException/*' />
     [Serializable]
-    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
Just curious, could this be because of a different version of the System.Data? according to [System.Data Namespace](https://apisof.net/catalog/System.Data) .netcore 2.1 and 3.1 each has a different version and a different pulic key.

Reply:
> The TypeForwardedFromAttribute attribute enables the target framework provider to trace a loaded type back to the original assembly it was forwarded from.

We don't mean to trace this type (SqlException) back to System.Data, that's why this tag itself doesn't qualify for Microsoft.Data.SqlClient.